packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -0,0 +1,112 @@
+
+* You've hit a rate limit when you're in the Admin Console, and a banner appears to inform you. Click **utilization history** in the banner to open the Rate Limits Dashboard for that particular endpoint or **syslog** to open the System Log.
+
+![Access the dashboard by link in the banner](/img/RLDashboard2.png "Displays the banner in the dashboard that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
+
+* You've hit a rate limit warning or violation, and a rate limit event (violation or warning) is created in the System Log from where you can click **View Utilization** under **Event info** to go to the rate limits dashboard. All the rate limit events in the System Log (violations and warnings) have a link to the rate limit dashboard.
+
+![Access the dashboard by link in the System Log](/img/RLDashboard3.png "Displays the event in the System Log that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
+
+## Key components of the dashboard
+
+The rate limits dashboard provides data usage information for the endpoint based on a set of statistics and shows multiple views of the data usage. The dashboard helps you to investigate the rate limit or violation issue and to prevent subsequent rate limit issues from occurring. To view the most recent data, refresh your screen. There may be a slight delay of two to four minutes in obtaining new data.
+
+### Overview pane
+
+The Overview pane provides a summary of the data use for the specified endpoint. The following information on data use is displayed:
+
+* Rate Limit: The rate limit for this bucket per minute for your org. If the traffic hits the rate limit for your org, the system triggers a rate limit violation.
+* Latest Utilization: The latest use for the bucket, which includes the endpoint and its sub-endpoints, as of the specified time
+* Average Utilization: The average use of the bucket within the last 24 hours and the last hour
+* Downtime: The amount of time that the org is affected by a rate limit violation on this endpoint
+
+### Rate limit multiplier
+
+A customer can additionally purchase the rate limit multiplier, which is an add-on feature that allows the customer to multiply an existing rate limit to, for example, three times the current rate limit.
+
+![Summary of the rate limit utilization over time pane](/img/RLDashboard4.png "Displays the line graph, bar graph, and timeline graph")
+
+The key benefit of purchasing the rate limit multiplier, therefore getting an increased rate limit, is not to stop end users from using an application. The rate limit multiplier is based on the customer's org and what the expected usage would be. If a customer has dynamic scale, the info icon beside **Rate Limit** in the Overview pane indicates which tier was purchased, and the graphs in the [Rate limit utilization over time pane] will reflect the applied rate limit multiplier.
```

Comment:
Are we purposely keeping " The rate limit multiplier is based on the customer's org and what the expected usage would be" vague? It's clear in the next line that Dynamic Scale is something that is purchased with multiple tiers. It's not clear here what the Workforce Multiplier is based on. Do we have the Workforce Multiplier documented somewhere?

Reply:
Yes, it's on purpose. The discussion in the Google doc with Michael and Joe was to keep this generic.